{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000100967598 chromedriver + 4973976\n1   chromedriver                        0x000000010095e913 chromedriver + 4938003\n2   chromedriver                        0x000000010051bcd6 chromedriver + 470230\n3   chromedriver                        0x000000010055f2ff chromedriver + 746239\n4   chromedriver                        0x000000010055f591 chromedriver + 746897\n5   chromedriver                        0x00000001005a1384 chromedriver + 1016708\n6   chromedriver                        0x00000001005839ed chromedriver + 895469\n7   chromedriver                        0x000000010059e856 chromedriver + 1005654\n8   chromedriver                        0x0000000100583793 chromedriver + 894867\n9   chromedriver                        0x000000010055121f chromedriver + 688671\n10  chromedriver                        0x000000010055256e chromedriver + 693614\n11  chromedriver                        0x0000000100921bb7 chromedriver + 4688823\n12  chromedriver                        0x0000000100926a51 chromedriver + 4708945\n13  chromedriver                        0x000000010092d40f chromedriver + 4736015\n14  chromedriver                        0x000000010092791a chromedriver + 4712730\n15  chromedriver                        0x00000001008fa87c chromedriver + 4528252\n16  chromedriver                        0x0000000100946f28 chromedriver + 4841256\n17  chromedriver                        0x00000001009470a7 chromedriver + 4841639\n18  chromedriver                        0x000000010095828f chromedriver + 4911759\n19  libsystem_pthread.dylib             0x00007fff204068fc _pthread_start + 224\n20  libsystem_pthread.dylib             0x00007fff20402443 thread_start + 15","children":[{"name":"User is able to login,this should fail","uid":"e68d66859d770ad7","parentUid":"e58ed9e739dab7be30e9f373a264212b","status":"broken","time":{"start":1684825090955,"stop":1684825133127,"duration":42172},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'chrome'"],"tags":["chrome","@pytest.mark.usefixtures('setup')"]}],"uid":"e58ed9e739dab7be30e9f373a264212b"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff91be2290>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is not allowed to enter alpha characters","uid":"8b39e24d4af6846d","parentUid":"78f6dedd569b6c0fbf99534fff57c179","status":"broken","time":{"start":1684826531272,"stop":1684826531272,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'chrome'"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"78f6dedd569b6c0fbf99534fff57c179"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffacfe5e10>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is not allowed to enter invalid phone number","uid":"340d82bf29c6403f","parentUid":"4465a8910ed4bd82a4f1f3ff344fc327","status":"broken","time":{"start":1684826531272,"stop":1684826531272,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'firefox'"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"4465a8910ed4bd82a4f1f3ff344fc327"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffab67a190>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is able to login,this should fail","uid":"db70a8e855bca114","parentUid":"123eee694e2fc915d29bb88c66b874a1","status":"broken","time":{"start":1684826531273,"stop":1684826531273,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'firefox'"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"123eee694e2fc915d29bb88c66b874a1"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff7bd75cd0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is not allowed to enter invalid phone number","uid":"bae256c4470baaa5","parentUid":"46cc16f5aa30689f393832966f1b5eca","status":"broken","time":{"start":1684826531275,"stop":1684826531275,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'chrome'"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"46cc16f5aa30689f393832966f1b5eca"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff7ebae210>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is able to login,this should fail","uid":"b8bd5bd1039e3070","parentUid":"4924af60bf29e7fa542cad3d23bb6110","status":"broken","time":{"start":1684826531275,"stop":1684826531275,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'chrome'"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"4924af60bf29e7fa542cad3d23bb6110"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff7f23ded0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is not allowed to enter alpha characters","uid":"daaf7e82af56579e","parentUid":"fb7ab00a7fe3f007ea52f429a7a78f6f","status":"broken","time":{"start":1684826531276,"stop":1684826531276,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'firefox'"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"fb7ab00a7fe3f007ea52f429a7a78f6f"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffb8705dd0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is able to change country code","uid":"1a41b62e600cd598","parentUid":"a535522c2af05625cb356a33a4456915","status":"broken","time":{"start":1684826531278,"stop":1684826531278,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'firefox'","{'country': 'Afghanistan', 'code': '+93'}"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"a535522c2af05625cb356a33a4456915"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffa9eae190>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is able to change country code","uid":"75ec73be64fef1d2","parentUid":"60ff9b4c9170f5751251c3903ae0e428","status":"broken","time":{"start":1684826531281,"stop":1684826531281,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["'chrome'","{'country': 'Afghanistan', 'code': '+93'}"],"tags":["@pytest.mark.usefixtures('setup')"]}],"uid":"60ff9b4c9170f5751251c3903ae0e428"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}